src/scss/sections/_testimonials-carousel.scss

```diff
@@ -1,3 +1,33 @@
 .testimonials-carousel {
-  background-color: $gray-100;
-}
+    &__quote::before {
```

Comment:
please don't duplicate properties, let's have first quote:before, quote:after together with the common style and then each of them with their single styles